CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 
 **Bugfixes**
 - ([[#419](https://github.com/ramsayleung/rspotify/issues/419)) Base64url encode instead of plain base64 encode for PKCE challenge code.
+- ([[#421](https://github.com/ramsayleung/rspotify/issues/421)) Filter `null`s on `tracks_features` requests
```

Comment:
Cool, thanks for your contribution. I think this PR should be marked as **Breaking Change**, because the type of `audio_features` change from `Vec<AudioFeatures>` to `Vec<Option<AudioFeatures>>`.

Everything else look good to me.

Reply:
Gotcha